spotbugs-tests/src/test/java/edu/umd/cs/findbugs/ba/Issue688Test.java

```diff
@@ -18,24 +18,17 @@
  */
 package edu.umd.cs.findbugs.ba;
 
-import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
-import static org.hamcrest.MatcherAssert.assertThat;
 
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 import edu.umd.cs.findbugs.AbstractIntegrationTest;
-import edu.umd.cs.findbugs.SortedBugCollection;
-import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
-import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
 
 class Issue688Test extends AbstractIntegrationTest {
 
     @Test
     void testIssue() {
-        performAnalysis("ghIssues/Issue688.class");
-        final BugInstanceMatcher bugTypeMatcher = new BugInstanceMatcherBuilder().build();
-        SortedBugCollection bugCollection = (SortedBugCollection) getBugCollection();
-        assertThat(bugCollection, containsExactly(0, bugTypeMatcher));
+        Assertions.assertDoesNotThrow(() -> performAnalysis("ghIssues/Issue688.class"));
```

Comment:
There is no issue 688, but this file was introduced in https://github.com/spotbugs/spotbugs/pull/688. Which is about the analysis throwing exception.